backend/prisma/schema.prisma

```diff
@@ -69,7 +70,40 @@ model Plan {
   specialEventType EventType
   createdAt        DateTime         @default(now())
   updatedAt        DateTime         @updatedAt
-  PlanMembership   PlanMembership[]
+  planMembership   PlanMembership[]
+  contact          Contact?          @relation(fields: [contactID], references: [id])
+  contactID        String?
+  FinalGift        FinalGift?
+}
+
+model Contact {
+  id                 String                    @id @default(uuid())
+  email              String?
+  firstName          String
+  lastName           String?
+  phoneNumber        String?
+  notes              String?
+  source             String?
+  ownerID            String
+  owner              User                      @relation(fields: [ownerID], references: [id])
+  importantDateEvent ImportantDateEvent[]
+  relationships      UserContactRelationship[]
+  Plan               Plan[]
+}
+
+model ImportantDateEvent {
+  id        String  @id @default(uuid())
+  contact   Contact @relation(fields: [contactID], references: [id])
+  contactID String
+  date      String
+  eventType String
+}
+
+model UserContactRelationship {
```

Comment:
isn't this model missing which user the contact has a relationship with?

Reply:
This should be renamed to `ContactRelationship`, i think the name is misleading. This model exists to store each contact relationships
![image](https://user-images.githubusercontent.com/106122976/233492611-dd7391b5-bb48-4cc7-bac8-d5eecae8250f.png)

Reply:
Changed the name